src/properties.jl

```diff
@@ -246,39 +246,42 @@ rmatrices = Dict(
 
 #---------------------- Equivariant matrices
 
-struct EuclideanMatrix{T} <: AbstractProperty where T<:Real
-   val::SMatrix{3, 3, Complex{T}, 9}
+struct EuclideanMatrix{T} <: AbstractProperty
+   val::SMatrix{3, 3, T, 9}
 end
 
 Base.show(io::IO, φ::EuclideanMatrix) = 
```

Comment:
I'm not sure about this stype of printing; it suggests 3 vectors rather than one matrix. Would it be ok to print it similarly to the spherical matrix, but with an "e" label instead of a "y" label? (yes, I also don't like the "m" label)

Reply:
Done! But I am wondering whether we should have a different label than the label of Euclidean vectors (since EuclideanMatrix satisfies a different equivariance property)

Reply:
I think it can be clear from the context that it is a matrix. If it were e and then three vectors then I would be confused.